understanding/20/captions-prerecorded.html

```diff
@@ -79,9 +78,7 @@ <h2>Examples of Captions (Prerecorded)</h2>
             
             <p>for the likes of sailors, soldiers and woodsmen.." </p>
             
-            <p>From Sample Transcript Formatting by Whit Anderson.</p>
-            
-         </li>
+            <p>From Sample Transcript Formatting by Whit Anderson.</dd>
```

Comment:
looks like some errant changes here?  maybe the file didnt' get fully committed?